modules/main.tf

```diff
@@ -70,35 +117,45 @@ resource "spacelift_module" "spacelift-private-workerpool" {
   name               = "spacelift-private-workerpool"
   terraform_provider = "aws"
   administrative     = false
-  branch             = "main"
+  branch             = "ibcdpe-1007-monitoring"
   description        = "Module for the spacelift private workerpool helm chart which deploys the K8s operator"
   repository         = "eks-stack"
   project_root       = "modules/spacelift-private-worker"
   space_id           = "root"
+
 }
 
 resource "spacelift_version" "spacelift-private-workerpool-version" {
   module_id      = spacelift_module.spacelift-private-workerpool.id
-  version_number = "0.2.0"
+  version_number = "0.2.1"
+  keepers = {
+    "version" = "0.2.1"
+  }
 }
 
-resource "spacelift_module" "spacelift-private-workerpool" {
+resource "spacelift_module" "spacelift_modules" {
+  for_each = local.spacelift_modules
+
   github_enterprise {
-    namespace = "Sage-Bionetworks-Workflows"
-    id        = "sage-bionetworks-workflows-gh"
+    namespace = each.value.github_enterprise.namespace
+    id        = each.value.github_enterprise.id
   }
 
-  name               = "spacelift-private-workerpool"
-  terraform_provider = "aws"
-  administrative     = false
-  branch             = "ibcdpe-935-vpc-updates"
-  description        = "Module for the spacelift private workerpool helm chart which deploys the K8s operator"
-  repository         = "eks-stack"
-  project_root       = "modules/spacelift-private-worker"
-  space_id           = "root"
+  name               = each.value.name
+  terraform_provider = each.value.terraform_provider
+  administrative     = each.value.administrative
+  branch             = each.value.branch
```

Comment:
Can this be used instead of hard coding the branch name (see comment above)

Reply:
Yes - This is the approach I am planning when I clean up the definition of tf resources